draft-ietf-tls-tls13.md

```diff
@@ -2201,9 +2204,13 @@ hash
 
 signature
 : This field indicates the signature algorithm that may be used.
-  The values indicate anonymous signatures, RSASSA-PKCS1-v1_5
-  {{RFC3447}}, DSA {{DSS}}, and ECDSA {{ECDSA}}, respectively.  The
-  "anonymous" value is meaningless in this context but used in
+  The values indicate anonymous signatures, RSASSA-PKCS-v1_5,
+  {{RFC3447}}, DSA {{DSS}}, ECDSA {{ECDSA}}, and
+  RSASSA-PSS respectively. Because all RSA signatures used in the
+  TLS protocol itself (as opposed to those in certificates)
+  are RSASSA-PSS, the "rsa" value refers solely to signatures
+  which appear in certificates.
+  The "anonymous" value is meaningless in this context but used in
   {{server-key-share}}.  It MUST NOT appear in this extension.
```

Comment:
In the “Remove DSA signature” PR (#235), I added deprecation for the “anonymous” value based on Dave’s advice…